packages/components/menu/src/menu-item.tsx

```diff
@@ -71,18 +87,23 @@ export const MenuItem = forwardRef<MenuItemProps, "div">(
       children,
       closeOnSelect: customCloseOnSelect,
       command,
+      disabled,
+      focusable,
       icon,
       isDisabled,
       isFocusable,
       ...props
     },
     ref,
   ) => {
+    disabled ??= isDisabled
+    focusable ??= isFocusable
+
     const {
       closeOnSelect,
       focusedIndex,
-      isNested,
-      isOpen,
+      isNested: nested,
```

Comment:
https://github.com/yamada-ui/yamada-ui/pull/3693/files#r1867394638

Reply:
Fixed in the commit [`06e6781` (#3693)](https://github.com/yamada-ui/yamada-ui/pull/3693/commits/06e6781b87f82567d837cec6ee04ab4a8714a944) !